dev/validate_bom__test.py

```diff
@@ -500,22 +517,28 @@ def wait_on_service(self, service_name, port=None, timeout=120):
     # I've only seen this happen once.
     time.sleep(1)
 
+    threadid = hex(threading.current_thread().ident)
```

Comment:
the id's are very large. Or at least mine are. hex provides a larger alphabet to make them differ from each other a little more noticably on inspection.